pkg/subnet/public.go

```diff
@@ -631,6 +641,8 @@ func (d *PublicDeployer) issueAddPermissionlessValidatorTX(
 	startTime uint64,
 	endTime uint64,
 	wallet primary.Wallet,
+	delegationFee uint32,
+	popBytes []byte,
```

Comment:
I now believe probably there is no need for json marshall/unmarshall and this can just receive a signer.ProofOfPossesion. The caller can generate that by filling the public fields PublicKey and ProofOfPosession, and then calling Verify()